src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -169,7 +170,11 @@ public void close(Collection<TopicPartition> partitions) {
                 log.error("Error closing writer for {}. Error: {}", tp, e.getMessage());
             }
         }
-
+        try {
+            kustoIngestClient.close();
```

Comment:
is it need to close the client? is it documented?

Reply:
Talked F2F - its a new feature in the client - to close timers